src/main/java/nc/students/ayaz/Video.java

```diff
@@ -44,4 +24,19 @@ void addComment(Comment comment) {
     void deleteComment(Comment comment) {
         comments.remove(comment);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
&#x1F34E; **Issue**
Why cant we remove this row? If we have same objects then equals bellow will return true.